src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig

```diff
@@ -132,9 +135,21 @@
                     {{ form_widget(form.step6) }}
                     <div class="text-right">
                         <a class="btn btn-primary btn-prev">{{ trans('PREVIOUS') }}</a>
+                        {% if hookcount('displayAdminProductsExtra') > 0 %}
+                            <a class="btn btn-primary btn-next">{{ trans('CONTINUE') }}</a>
+                        {% endif %}
                         <input type="submit" id="submit" value="{{ trans('SAVE') }}" class="btn btn-warning" />
                     </div>
                 </div>
+                {% if hookcount('displayAdminProductsExtra') > 0 %}
+                <div role="tabpanel" class="tab-pane panel panel-default" id="hooks">
+                    {{ renderhook('displayAdminProductsExtra') }}
+                    <div class="text-right">
+                        <a class="btn btn-primary btn-prev">{{ trans('PREVIOUS') }}</a>
+                        <input type="submit" id="submit" value="{{ trans('SAVE') }}" class="btn btn-warning" />
```

Comment:
Hi !
At this point, we follows (indirectly) the Symfony good practices by using the terms of Symfony's TranslatorInterface. Even if THIS class does not implements TranslatorInterface, we follow it, and we use our service (injected in the constructor) that implements this interface. So I think keeping the same method name (and everywhere in the app) is a best practice.